### ISSUE TITLE
Question of normalization parameters used for feature extraction?

### ISSUE THREAD

**Issue by GWwangshuo** [dazhangyu123/ACMIL#3]
Thanks for your nice work. I would like to know in what way dino_vit_small_patch16_ep200.torch goes through the normalization of the training images because in practice we often put the test images through the same normalization process as the training set, e.g.

transforms_beit3 = transforms.Compose([
    transforms.Resize((224,224), interpolation=3), transforms.
    transforms.ToTensor(),
    transforms.Normalize(mean= (0.5, 0.5, 0.5), std= (0.5, 0.5, 0.5))
])

trnsfrms_CTransPath_Brow_iBOTViT = transforms.Compose(
[
    transforms.Resize((224,224)),
    transforms.ToTensor(),
    transforms.Normalize(mean = (0.485, 0.456, 0.406), std = (0.229, 0.224, 0.225))
]
)

**Comment by dazhangyu123** [comment]
We are using the second option. Thank you for bringing this to our attention. Based on your question, I noticed that the dataset_h5.py file was missing, and I have added it to [datasets/dataset_h5.py](https://github.com/dazhangyu123/ACMIL/blob/main/datasets/dataset_h5.py). More details can be found in this file.
